### PR TITLE
Use named exports rather than default exports when relevant

### PR DESCRIPTION
**Pull request by antca**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -6,28 +6,15 @@ Promise.config({
...
-import stores from './stores';
+export { default as Action } from './Action';
+export { default as actions } from './actions';
+export { default as deps } from './deps';
+export { default as Flux } from './Flux';
+export { default as preparable } from './preparable';
+export { default as prepare } from './prepare';
+export { default as root } from './root';
+export { default as Store } from './Store';
+export { default as stores } from './stores';
```
Comment (eTorAken) [inline lib/index.js]: empty line wanted?

**Comment by eTorAken** [comment]
You are good to go :+1:

**Comment by eTorAken** [comment]
Ping me when rebased please

**Comment by antca** [comment]
Rebased @eTorAken

**Pull request merged**